### PR TITLE
Feature/UI gateway seperation(#KHarsha)

### PR DESCRIPTION
**Pull request by raxkumar-wedaa**
- Created a Gateway Modal 
- Modified UI modal with application name and port number check 
- Connection between UI to Service,UI to Gateway and Gateway to Service is set by default rest
- No connection from other node to UI and UI to Db

### CODE REVIEW

File: src/components/Modal/GatewayModal.js [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+        [column]: value,
+        serverPort: value,
+      }));
+    } else {
+      setApplicationData((prev) => ({
+        ...prev,
+        [column]: value,
+      }));
+    }
+    if (column === "serverPort" && ApplicationData.serverPort === "9000") {
```
Comment (craxkumar) [inline src/components/Modal/GatewayModal.js:118]: No need of the default value

File: src/components/Modal/GatewayModal.js [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+            {PortNumberError && (
+              <Alert
+                status="error"
+                padding="4px"
+                fontSize="12px"
+                borderRadius="3px"
+                mb={2}
+              >
+                <AlertIcon style={{ width: "14px", height: "14px" }} />
+                Port Number already exists. Please choose a unique Number.
```
Comment (craxkumar) [inline src/components/Modal/GatewayModal.js:299]: unique port or unique port number.
but not 
unique number.

**Review by craxkumar — CHANGES_REQUESTED** [review]

File: src/components/Modal/GatewayModal.js [hunk 3]
```diff
@@ -0,0 +1,390 @@
...
+      setDuplicateApplicationNameError(true);
+      return false;
+    } else {
+      setDuplicateApplicationNameError(false);
+      return true;
+    }
+  };
+
+  //check whether port number is unique and lies within the range
+  const ValidatePortNumber = (value) => {
```
Comment (raxkumar-wedaa) [inline src/components/Modal/GatewayModal.js:74]: method/function names should be camel casing always 
validatePortNumber !!

**Review by raxkumar-wedaa — COMMENTED** [review]
Please resolve !

**Review by rohith-wedaa — APPROVED** [review]

**Pull request merged**